slimish_jinja/parse.py

```diff
@@ -39,6 +39,12 @@ def parse(self):
         except StopIteration:
             pass
 
+    def get_next_token(self):
```

Comment:
Instead of this, update the lexer class to have `__next__()` , which is the equivalent of py2 `next()`